### PR TITLE
[13.0][IMP] base_maintenance: Add a Maintenance Request Report

### PR DESCRIPTION
**Pull request by marcelsavegnago**

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM

**Comment by etobella** [comment]
Review Travis linting

**Comment by etobella** [comment]
@marcelsavegnago Can you try to install locally the pre-commit and then apply the changes? This way it will resolve automatically your linting problems. It will only works on 13.0 and over, but it is a huge step.

https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97
https://github.com/OCA/maintainer-quality-tools#qa-tools-for-odoo-maintainers-mqt

**Comment by marcelsavegnago** [comment]
@etobella Done. Thank you for your help and I'm sorry for doing this just now. For the past few days I have been working on client projects and Brazilian localization modules and unfortunately the time is short: D

**Comment by marcelsavegnago** [comment]
@dalonsod @lmiguens-solvos Can you review this PR ? please :)

File: base_maintenance/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1 +1,2 @@
-* Lois Rilo <lois.rilo@forgeflow.com>
+* Lois Rilo <lois.rilo@eficent.com>
```
Comment (dalonsod) [inline base_maintenance/readme/CONTRIBUTORS.rst]: Maybe @LoisRForgeFlow does not agree with this change XD

File: base_maintenance/views/report_maintenance_request.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                            <p class="m-0" t-field="doc.stage_id" />
+                        </div>
+                    </div>
+                    <div class="row mt32 mb32" id="equipament_informations">
+                        <div t-if="doc.category_id" class="col-auto mw-100 mb-2">
+                            <strong>Category:</strong>
+                            <p class="m-0" t-field="doc.category_id" />
+                        </div>
+                        <div t-if="doc.equipment_id" class="col-auto mw-100 mb-2">
+                            <strong>Equipament:</strong>
```
Comment (dalonsod) [inline base_maintenance/views/report_maintenance_request.xml]: It's good time to fix this typo:
```suggestion
                            <strong>Equipment:</strong>
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only cosmetic changes

File: base_maintenance/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (marcelsavegnago) [reply base_maintenance/readme/CONTRIBUTORS.rst]: @dalonsod So, what do you think of not changing that? Or better already change?

Comment (marcelsavegnago) [reply base_maintenance/readme/CONTRIBUTORS.rst]: @LoisRForgeFlow What do you think ?

Comment (etobella) [reply base_maintenance/readme/CONTRIBUTORS.rst]: I can answer for him, eficent was the name of the company, but it has recently changed for forgeflow, so keep forgeflow please :wink:

Comment (dalonsod) [reply base_maintenance/readme/CONTRIBUTORS.rst]: @marcelsavegnago don't forget to undo this change

**Review by jarroyomorales — APPROVED** [review]
LGTM!

**Comment by jarroyomorales** [comment]
@dalonsod Can you check again so it can get merged?

**Comment by dalonsod** [comment]
@marcelsavegnago `CONTRIBUTORS.rst` file is still wrong, as @etobella said https://github.com/OCA/maintenance/pull/97#discussion_r475083583

**Comment by marcelsavegnago** [comment]
@dalonsod done.

**Review by dalonsod — APPROVED** [review]

**Comment by etobella** [comment]
Just one last comment, con you change the commit message of the last commit, I know this is purely visual, but it is in order to maintain some homogeneity. Then we can proceed with the merge :wink:

**Comment by marcelsavegnago** [comment]
@etobella Done.

**Pull request merged**